gui-agent/vmside.c

```diff
@@ -2368,7 +2363,6 @@ int main(int argc, char **argv)
         } while (busy);
 
     }
-    return 0;
```

Comment:
Please don't remove final return statement from non-void function, even if right now it's unreachable.